### ISSUE TITLE
upgrading to 5.39 fails due to bleve

### ISSUE THREAD

**Issue by shurrman** [mattermost/docker#38]
I had 5.38.2 up and running, no issues. Decided to upgrade to 5.39 as it was called a "quality release"... ;-) (well, I always prefer to be on the bleeding edge of technologies).
Performed "down" with compose, copied full directory where MTM lives just to back it up, changed tag to 5.39.0 in the .env, performed "up -d" with compose.
It got stuck on "starting bleve" (as per logs). I put it down back, and disabled bleve in config.json.
After that MTM started ok. I went to system console and enabled bleve back again. Dropped indexes, now indexing...

**Comment by mrckndt** [comment]
Hi @shurrman, is this problem still persistent with newer releases? I tried this in September and now with different versions and i can't reproduce the seen issue. Looks like a one-time hiccup to me.

**Comment by shurrman** [comment]
@mrckndt Just recently I upgraded to 6.2. Accompanied with the upgrade, I was moving MTM to the new server. So, to reduce copying time and size, I disabled Bleve and deleted the index. After I started on the new server - Bleve turned out to be on, and reindexing took place (which is a bit odd... but I am not intended to reproduce it), while other checkboxes (use Bleve for search and highlights) were off. 
So yes, probably it is gone.
